src/Symfony/Component/Workflow/Registry.php

```diff
@@ -23,19 +24,26 @@ class Registry
 
     /**
      * @param Workflow $workflow
-     * @param string   $className
+     * @param SupportStrategyInterface $supportStrategy
      */
-    public function add(Workflow $workflow, $className)
+    public function add(Workflow $workflow, SupportStrategyInterface $supportStrategy)
```

Comment:
It's the standard way of introducing a BC layer in symfony. It's kinda like method overloading